packages/common/react-components/src/components/Avatar/Avatar.stories.tsx

```diff
@@ -13,7 +13,15 @@ export default {
 
 export const Default = {
   args: {
-    fallbackValue: '20970b563fc49b5bb194a6ffdff376031a3a11f9481360c071c3fed87874106b',
-    label: <span className='sr-only'>Hello</span>
+    fallbackValue: 'squirrel',
```

Comment:
if possible we've tried to avoid hardcoding foo/bar-like example values in favour of slightly more realistic values, potentially indicating a practical use case

Reply:
Agreed, I think I might’ve committed this particular value by mistake.